src/server/ApiRoutes/RouteMethods.js

```diff
@@ -0,0 +1,10 @@
+import appConfig from '@appConfig';
+import Bib from './Bib';
+import Search from './Search';
+import Hold from './Hold';
+
+export default {
+  bib: Bib.bibSearchAjax,
+  search: Search.searchAjax,
+  holdRequest: Hold.newHoldRequestAjax,
```

Comment:
All of this makes sense but I would remove the `Ajax` from the function names since now you're using them on the server and client.

Reply:
Removed for `bib` and `holdRequest`, but there is already a `search` method in `Search`. Open to suggestions on naming.